nixos/modules/services/misc/gitlab.nix

```diff
@@ -88,7 +88,7 @@ let
           wiki = true;
           snippets = true;
           builds = true;
-          container_registry = true;
+          container_registry = cfg.registry.enable;
```

Comment:
This affects new gitlab project defaults, about whether the registry should be enabled or not.

I assume people want to have a registry around, but not flip that default. Maybe introduce a second config options, which defaults to the value of `cfg.registry.enable`?